### DISCUSSION TITLE
How to configure CFtool for CPeditor in macOS

### DISCUSSION THREAD

**Discussion by dhimanda** [cpeditor/cpeditor#803] [Q&A]
Hi, I am a new mac user. I used CPeditor in windows. Recently I changed my PC to Macbook Air.  
Can you please tell me the way, how to connect cpEditor+cfTool in macOS?

**Comment by ouuan** [comment]
What problem did you meet? Do you have problems installing/configuring CF Tool, or CF Tool works but doesn't connect with CP Editor?
